airbyte-integrations/connectors/source-chargebee/source_chargebee/schemas/credit_note.json

```diff
@@ -4,453 +4,863 @@
+      "type": [
+        "integer",
+        "null"
+      ],
       "minimum": 0
     },
     "amount_refunded": {
-      "type": ["integer", "null"],
+      "type": [
+        "integer",
+        "null"
+      ],
       "minimum": 0
     },
     "amount_available": {
-      "type": ["integer", "null"],
+      "type": [
+        "integer",
+        "null"
+      ],
       "minimum": 0
     },
     "refunded_at": {
-      "type": ["integer", "null"]
+      "type": [
+        "integer",
+        "null"
+      ]
     },
     "voided_at": {
-      "type": ["integer", "null"]
+      "type": [
+        "integer",
+        "null"
+      ]
     },
     "generated_at": {
-      "type": ["integer", "null"]
+      "type": [
+        "integer",
+        "null"
+      ]
     },
     "resource_version": {
-      "type": ["integer", "null"]
+      "type": [
+        "integer",
+        "null"
+      ]
     },
     "updated_at": {
-      "type": ["integer", "null"]
+      "type": [
+        "integer",
+        "null"
+      ]
     },
     "sub_total": {
-      "type": ["integer", "null"],
+      "type": [
+        "integer",
+        "null"
+      ],
       "minimum": 0
     },
     "sub_total_in_local_currency": {
-      "type": ["integer", "null"],
+      "type": [
+        "integer",
+        "null"
+      ],
       "minimum": 0
     },
     "total_in_local_currency": {
-      "type": ["integer", "null"],
+      "type": [
+        "integer",
+        "null"
+      ],
       "minimum": 0
     },
     "local_currency_code": {
-      "type": ["string", "null"]
+      "type": [
+        "string",
+        "null"
+      ]
     },
     "round_off_amount": {
-      "type": ["integer", "null"],
+      "type": [
+        "integer",
+        "null"
+      ],
       "minimum": 0
     },
     "fractional_correction": {
-      "type": ["integer", "null"],
+      "type": [
+        "integer",
+        "null"
+      ],
       "minimum": 0
     },
     "deleted": {
-      "type": ["boolean", "null"]
+      "type": [
+        "boolean",
+        "null"
+      ]
     },
     "create_reason_code": {
-      "type": ["string", "null"]
+      "type": [
+        "string",
+        "null"
+      ]
     },
     "vat_number_prefix": {
-      "type": ["string", "null"],
+      "type": [
+        "string",
+        "null"
+      ],
       "maxLength": 10
     },
+    "base_currency_code": {
+      "type": [
+        "string",
+        "null"
+      ],
+      "maxLength": 3
+    },
+    "business_entity_id": {
+      "type": [
+        "string",
+        "null"
+      ],
+      "maxLength": 50
+    },
+    "channel": {
+      "type": [
+        "string",
+        "null"
+      ]
+    },
+    "exchange_rate": {
+      "type": [
+        "integer",
+        "null"
+      ]
+    },
+    "is_digital": {
+      "type": [
+        "boolean",
+        "null"
+      ]
+    },
+    "object": {
+      "type": [
+        "string",
+        "null"
+      ]
+    },
+    "is_vat_moss_registered": {
```

Comment:
Please verify this field.

Reply:
Hi @pnilan, we can collect data for this field, but it depends on the record, in some cases we have it and in others not. That's why i've removed the field from the credit note schema, and added "additionalProperties" to true to include it anyway. It's good for you like this ?

Reply:
@KimPlv Thanks for the confirmation. I confirmed with Chargebee support that this specific field is only available for accounts that are configured for EU taxes, which is why it wasn't populating on my end. I think we can go ahead and include it in the schema and add a `$comment` property to note this.

I will update this on my end and verify the tests are running properly and then we should be able to merge the PR. I will have to merge this as a separate PR though for our CI pipeline to run properly. I'll let you know when I have the PR submitted.

Reply:
@pnilan fine ! Thanks for the information. I've made the change for this field